utils/_context/_scenarios/__init__.py

```diff
@@ -586,6 +586,17 @@ class _Scenarios:
         doc="",
     )
 
+    apm_tracing_efficient_payload = EndToEndScenario(
```

Comment:
Can you add this in `.github/workflows/run-end-to-end.yml` ? 

Sorry, it's a big boilerplate, but so far, github actions can't do dynamic steps